cmd/soroban-rpc/internal/db/db.go

```diff
@@ -23,35 +22,24 @@ var migrations embed.FS
 var ErrEmptyDB = errors.New("DB is empty")
 
 const (
-	ledgerEntriesTableName      = "ledger_entries"
 	metaTableName               = "metadata"
 	latestLedgerSequenceMetaKey = "LatestLedgerSequence"
 )
 
-type DB interface {
-	NewLedgerEntryUpdaterTx(forLedgerSequence uint32, maxBatchSize int) (LedgerEntryUpdaterTx, error)
-	NewLedgerEntryReaderTx() (LedgerEntryReaderTx, error)
-	io.Closer
-}
-
-type LedgerEntryUpdaterTx interface {
-	UpsertLedgerEntry(key xdr.LedgerKey, entry xdr.LedgerEntry) error
-	DeleteLedgerEntry(key xdr.LedgerKey) error
-	Done() error
-}
-
-type LedgerEntryReaderTx interface {
-	GetLatestLedgerSequence() (uint32, error)
-	GetLedgerEntry(key xdr.LedgerKey) (bool, xdr.LedgerEntry, error)
-	Done() error
```

Comment:
Why have you removed the interfaces? They can be used for mocking in tests.

I was planning on adding a performance benchmark for the preflight endpoint later on, although ... for that we could maybe use a real on-disk db.

Reply:
Ah, I see. thanks!